source/extensions/clusters/redis/redis_cluster.h

```diff
@@ -211,6 +211,9 @@ class RedisCluster : public Upstream::BaseDynamicClusterImpl {
     bool enableRedirection() const override { return false; }
     uint32_t maxBufferSizeBeforeFlush() const override { return 0; }
     std::chrono::milliseconds bufferFlushTimeoutInMs() const override { return buffer_timeout_; }
+    Extensions::NetworkFilters::Common::Redis::Client::ReadPolicy readPolicy() const override {
+      return Extensions::NetworkFilters::Common::Redis::Client::ReadPolicy::Any;
```

Comment:
This is effectively not in used here.  Also this is configure for making the "Cluster Slots" calls, since we call cluster slots on both the master and slaves, ANY is more appropriate here.  The default for actual data plane on the listener is Master.

Reply:
can you add a comment here?

Reply:
Done